doc/cabal-commands.rst

```diff
@@ -1390,7 +1390,10 @@ A list of all warnings with their constructor:
 - ``unknown-arch``: unknown architecture in condition.
 - ``unknown-compiler``: unknown compiler in condition.
 - ``missing-bounds-important``: missing upper bounds for important dependencies (``base``, and for ``custom-setup`` ``Cabal`` too).
-- ``missing-upper-bounds``: missing upper bound in dependency (excluding test-suites and benchmarks).
+- ``missing-upper-bounds``: missing upper bound in dependency [#dep-excl]_.
```

Comment:
Excellent manual addition, but I don’t think the footnote is needed for ``missing-upper-bounds``

Reply:
Could you please explain further as to me that would seem inconsistent?

```diff
- excluding test-suites and benchmarks
+ In dependencies excluding test-suites and benchmarks.
```

Reply:
Sorry, I thought this pointed to the `Note` below that.

It is perfectly fine.